front/temporal/scrub_workspace/activities.ts

```diff
@@ -115,6 +124,37 @@ async function deleteAllConversations(auth: Authenticator) {
         const agentMessageIds = removeNulls(
           messages.map((m) => m.agentMessageId)
         );
+
+        const retrievalActions = await AgentRetrievalAction.findAll({
+          attributes: ["id"],
+          where: { agentMessageId: agentMessageIds },
+        });
+        const retrievalDocuments = await RetrievalDocument.findAll({
+          attributes: ["id"],
+          where: { retrievalActionId: retrievalActions.map((a) => a.id) },
+        });
+        await RetrievalDocumentChunk.destroy({
+          where: { retrievalDocumentId: retrievalDocuments.map((d) => d.id) },
+        });
+        await RetrievalDocument.destroy({
+          where: { retrievalActionId: retrievalActions.map((a) => a.id) },
```

Comment:
Shouldn't we use `retrievalDocuments` from above here?

Reply:
We fetch them to delete  the chunks, but in practice it's probably a tiny bit better to use `retrievalActionId` since the list will be a bit smaller :)